ports/atmel-samd/Makefile

```diff
@@ -268,6 +268,7 @@ SRC_COMMON_HAL = \
 	neopixel_write/__init__.c \
 	os/__init__.c \
 	storage/__init__.c \
+        supervisor/Status.c \
```

Comment:
`supervisor/Status.c` line should have a single tab, not spaces.

Reply:
argh! I meant to go back and fix that before pushing...